src/guide/4-initial-conditions/4-rng-seeds.rst

```diff
@@ -1,3 +1,41 @@
 Random Number Generation
 ========================
 
+Agent variables can be initialised with randomly generated data:
```

Comment:
This needs some discussion. E.g. do we want to push people towards creating agent populations inside init functions instead. This is (currently) required for use of ensembles.

Reply:
And long term, we want to add device init functions too as a more efficient alternative for big initial pops.